svo (trigger functions).xml

```diff
@@ -4624,70 +4624,83 @@ end
+  
+  function svo.valid.noshruggingbalance()
+    svo.checkaction(svo.dict.shrugging.physical)
+    if actions.shrugging_physical then
+      lifevision.add(actions.shrugging_physical.p, 'offbal')
+    end
+  end
+  
+  function svo.valid.gotshruggingbalance()
+    svo.checkaction(svo.dict.gotbalance.happened, true)
+    svo.dict.gotbalance.happened.tempmap[#svo.dict.gotbalance.happened.tempmap+1] = 'shrugging' -- hack to allow multiple balances at once
+    lifevision.add(actions.gotbalance_happened.p)
   end
-end
-
-function svo.valid.gotshruggingbalance()
-  svo.checkaction(svo.dict.gotbalance.happened, true)
-  svo.dict.gotbalance.happened.tempmap[#svo.dict.gotbalance.happened.tempmap+1] = 'shrugging' -- hack to allow multiple balances at once
-  lifevision.add(actions.gotbalance_happened.p)
-end
 end
 
 if svo.haveskillset('voicecraft') then
-function svo.valid.usedvoicebalance()
-  svo.checkaction(svo.dict.stolebalance.happened, true)
-  lifevision.add(actions.stolebalance_happened.p, nil, 'voice')
-end
-
-function svo.valid.gotvoicebalance()
-  svo.checkaction(svo.dict.gotbalance.happened, true)
-  svo.dict.gotbalance.happened.tempmap[#svo.dict.gotbalance.happened.tempmap+1] = 'voice' -- hack to allow multiple balances at once
-  lifevision.add(actions.gotbalance_happened.p)
-end
+  function svo.valid.usedvoicebalance()
+    svo.checkaction(svo.dict.stolebalance.happened, true)
+    lifevision.add(actions.stolebalance_happened.p, nil, 'voice')
+  end
+  
+  function svo.valid.gotvoicebalance()
+    svo.checkaction(svo.dict.gotbalance.happened, true)
+    svo.dict.gotbalance.happened.tempmap[#svo.dict.gotbalance.happened.tempmap+1] = 'voice' -- hack to allow multiple balances at once
+    lifevision.add(actions.gotbalance_happened.p)
+  end
 end
 
 if svo.haveskillset('terminus') then
-function svo.valid.usedwordbalance()
-  svo.checkaction(svo.dict.stolebalance.happened, true)
-  lifevision.add(actions.stolebalance_happened.p, nil, 'word')
-end
-
-function svo.valid.gotwordbalance()
-  svo.checkaction(svo.dict.gotbalance.happened, true)
-  svo.dict.gotbalance.happened.tempmap[#svo.dict.gotbalance.happened.tempmap+1] = 'word' -- hack to allow multiple balances at once
-  lifevision.add(actions.gotbalance_happened.p)
-end
+  function svo.valid.usedwordbalance()
+    svo.checkaction(svo.dict.stolebalance.happened, true)
+    lifevision.add(actions.stolebalance_happened.p, nil, 'word')
+  end
+  
+  function svo.valid.gotwordbalance()
+    svo.checkaction(svo.dict.gotbalance.happened, true)
+    svo.dict.gotbalance.happened.tempmap[#svo.dict.gotbalance.happened.tempmap+1] = 'word' -- hack to allow multiple balances at once
+    lifevision.add(actions.gotbalance_happened.p)
+  end
 end
 
 if svo.haveskillset('anathema') then
-function svo.valid.usedanathemabalance()
-  svo.checkaction(svo.dict.stolebalance.happened, true)
-  lifevision.add(actions.stolebalance_happened.p, nil, 'anathema')
+  function svo.valid.usedanathemabalance()
```

Comment:
Another comment so I know where to look when I get back at this, is all of this space formatting ok or something messed up? It looks fine and I'm not receiving mudlet errors so I guess nothing iffy happening here

Reply:
It's fine and actually better than before as this function declaration is only done if the anathema skillset is available.